db/migration/src/m20250219_000020_logs_add_index.rs

```diff
@@ -0,0 +1,47 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_index(
+                sea_query::Index::create()
+                    .if_not_exists()
+                    .name("idx_log_status_block_number_processed")
+                    .table(LogStatus::Table)
+                    .col((LogStatus::BlockNumber, IndexOrder::Asc))
+                    .col(LogStatus::Processed)
+                    .to_owned(),
+            )
+            .await
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_index(Index::drop().name("idx_log_status_block_number_processed").to_owned())
+            .await
+    }
+}
+
+#[derive(DeriveIden)]
```

Comment:
We should look whether this structure can be reimported from a previous migration step.

Reply:
What if some migration wants to add column?
It would make the code look weird - some migrations reimporting from others. And some having to redefine it, because they add columns.

That's a bad idea from code readability perspective IMHO.

Reply:
I agree.